plugins/rbac-backend/src/service/policies-rest-api.ts

```diff
@@ -47,6 +55,10 @@ export class PolicesServer {
     private readonly permissions: PermissionEvaluator,
     private readonly options: RouterOptions,
     private readonly enforcer: Enforcer,
+    private readonly config: Config,
+    private readonly logger: Logger,
+    private readonly discovery: PluginEndpointDiscovery,
+    private readonly pluginIdProvider: { getPluginIds: () => string[] },
```

Comment:
If you don't name the type here (through a named interface), why not using simply:
``` typescript
private readonly pluginIdProvider: () => string[],
```
?

Reply:
I would like interface more, than anonymous function.